source/includes/steps-install-shell-linux-deb.yaml

```diff
@@ -10,18 +10,18 @@ content: |
 
        wget -qO - https://www.mongodb.org/static/pgp/server-{+pgp-version+}.asc | sudo apt-key add -
```

Comment:
[question] When I run this command in my Ubuntu host, it comes up with the `apt-key` deprecation warning. Should we update this command?

Here's the warning I receive: 
`Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))`

Reply:
Thanks Ali. I discussed with dev and updated the commands to avoid the deprecation warning. I tested this on a Ubuntu 22.04 VM and seems to work 🚀

Reply:
Awesome thanks for updating this! Just had one more follow-up question: with the new command printing out the GPG key, is there anything that the user would have to do with it (e.g. copy-paste it somewhere, save it for future use, etc.) or is that more of like a verification?

Reply:
Got it! I guess with that (once Anna confirms), I'd suggest maybe a small note that mentions no action needs to be taken with the printed key, but this isn't a blocking suggestion.